core/amber/src/main/scala/edu/uci/ics/texera/web/resource/WorkflowWebsocketResource.scala

```diff
@@ -51,13 +54,18 @@ class WorkflowWebsocketResource extends LazyLogging {
     SessionState.setState(session.getId, sessionState)
     val wid = session.getRequestParameterMap.get("wid").get(0).toLong
     val cuid = session.getRequestParameterMap.get("cuid").get(0).toInt
+    var cuAccessEnum: PrivilegeEnum = null
+    if (KubernetesConfig.kubernetesComputingUnitEnabled) {
```

Comment:
can you change this logic to be:
- remove the if (KubernetesConfig.kubernetesComputingUnitEnabled)
- PrivilegeEnum.valueOf(session.getUserProperties.getOrElse(HeaderField.UserComputingUnitAccess, {// call the getComputingUnitAccess to query the database for the cu access}). 

KubernetesConfig.kubernetesComputingUnitEnabled this flag should just be a switch of having k8s pod as CU or no. It should NOT be used to determine the way of verifying CU access.